test/zdtm/static/cgroup_ifpriomap.checkskip

```diff
@@ -1,3 +1,9 @@
 #!/bin/bash
 
-! test -f /sys/fs/cgroup/cgroup.controllers
+test -f /sys/fs/cgroup/cgroup.controllers && exit 1
+
+grep -q 'cgroup rw,.*net_prio' /proc/mounts && exit 1
```

Comment:
why do we want to skip the test if the net_prio cgroup hierarchy is mounted?

Reply:
The test wants to mount the controller itself and if it is used by the system already the test fails.

Reply:
Mounting cgroup should be possible multiple times, maybe you can show real error you try to fix?